cmd/consts.go

```diff
@@ -0,0 +1,6 @@
+package cmd
+
+const (
```

Comment:
I wasn't sure if they should go into `root.go` since neither of the ones I put in `consts.go` are used in that file. Having a separate file is just something I've seen other projects do. I can certainly concede that having it in `root.go` is perfectly fine.